src/crimson/net/SocketConnection.cc

```diff
@@ -71,6 +71,7 @@ seastar::future<bool> SocketConnection::is_connected()
 
 seastar::future<> SocketConnection::send(MessageRef msg)
 {
+  logger().debug("{} --> {} === {}", messenger, get_peer_addr(), *msg);
```

Comment:
Just noticed async-msgr doesn't use connection prefix in this two cases. So I think it's OK not reuse the `SocketConnection::print()` here, if we want to keep the classic format.